apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -1,78 +1,35 @@
 module MittKonto.Main where
 
-import Prelude
+import Prelude (bind, const, discard, pure, when, ($), (<<<))
```

Comment:
I think `Prelude` should be imported fully

Reply:
The PS compiler expects only one import to be unspecified (which is usually the Prelude), so rather than specifying that one I would specify any other open imports there are.

Reply:
Ah ok then it must be because I was fixing imports all at once, but good to know!